src/openrct2/world/MapAnimation.cpp

```diff
@@ -35,185 +35,80 @@
 #include "tile_element/TrackElement.h"
 #include "tile_element/WallElement.h"
 
-using namespace OpenRCT2;
-
-using map_animation_invalidate_event_handler = bool (*)(const CoordsXYZ& loc);
-
-static std::vector<MapAnimation> _mapAnimations;
-
-constexpr size_t kMaxAnimatedObjects = 2000;
-
-static bool InvalidateMapAnimation(const MapAnimation& obj);
+#include <algorithm>
+#include <vector>
 
-static bool DoesAnimationExist(int32_t type, const CoordsXYZ& location)
-{
-    for (const auto& a : _mapAnimations)
-    {
-        if (a.type == type && a.location == location)
-        {
-            // Animation already exists
-            return true;
-        }
-    }
-    return false;
-}
+using namespace OpenRCT2;
 
-void MapAnimationCreate(int32_t type, const CoordsXYZ& loc)
-{
-    if (!DoesAnimationExist(type, loc))
-    {
-        if (_mapAnimations.size() < kMaxAnimatedObjects)
-        {
-            // Create new animation
-            _mapAnimations.push_back({ static_cast<uint8_t>(type), loc });
-        }
-        else
-        {
-            LOG_ERROR("Exceeded the maximum number of animations");
-        }
-    }
-}
+// Set to track tiles that contain animated elements
```

Comment:
not track tiles just any tiles.

Reply:
I think you've misread that one, its "to track tiles" as in "tracking tiles", unrelated to track elements.

Reply:
Changed that comment.